### ISSUE TITLE
Create Gallery

### ISSUE THREAD

**Issue by ashmitjsg** [Team-Abraxas/Abraxas-Website-2024#2]
Update the Gallery Page to showcase each year's images. It should have:

- A way to navigate to a specific year.
- Show the gallery of the year selected.

**Comment by kishankumarkk08** [comment]
11th February 2024 :
[](https://www.pinterest.com/pin/945685621727701326/)
[](https://www.pinterest.com/pin/63543044735906675/)
[](https://pin.it/7Ue5eznN0)
Designs for the gallery page.

**Comment by kishankumarkk08** [comment]
11th February 2024:
[Grid Based Gallery Page](https://in.pinterest.com/pin/945685621727701326/)
[Hexagonal Pattern Display](https://in.pinterest.com/pin/63543044735906675/)
[Hexagonal Pattern Display 2](https://in.pinterest.com/pin/56858014015352533/sent/?invite_code=d1fc70e1deb7449fa0f90a7ea61a91f6&sfo=1)

**Comment by ashmitjsg** [comment]
Search something a little bit animated. Instead of searching on Pinterest I will recommend browsing through websites.

**Comment by kishankumarkk08** [comment]
12th January 2024
[Kiko Wu](https://www.k-wu.com/)
[Manifesto](https://manifesto.itsoffbrand.com/)
[Thats Bettering](https://www.thatsbettering.com/)

**Comment by kishankumarkk08** [comment]
13th January 2024
[Zennsfoto](https://www.zennsfoto.de/)
[twodee](https://twodee.com.au/)
[Danielhedquist](https://www.danielhedquist.com/portfolio)

**Comment by ashmitjsg** [comment]
At least 10 sites a day for next three days. And search designs which you can code easily.

**Comment by kishankumarkk08** [comment]
[Workik](https://workik.com/best-gallery-designs-for-websites-with-html-css-and-javascript) (Contain Templates)
[Croing](https://croing.com/#/)
[Behance](https://www.behance.net/gallery/179820799/Citisum-Real-Estate-Dashboard-Design)
[UI Design Daily](https://www.uidesigndaily.com/)
We can take few figma templates from here also we can take the design of the entire website.

**Comment by kishankumarkk08** [comment]
[COPS IIT BHU](https://www.copsiitbhu.co.in/)
[Pexels](https://www.pexels.com/search/illustration/)
[E-Cell IIT Indore](http://ecell.iiti.ac.in/)
[E-Cell IIT Guwahati](https://www.ecelliitg.in/abt.html)
[E-Cell IIT Roorkee](https://ecelliitr.org/aboutus/gallery)

**Comment by ashmitjsg** [comment]
We will work on this: https://yansukim.com/

**Comment by kishankumarkk08** [comment]
Succesfully created the required transition.

**Comment by kishankumarkk08** [comment]
I was having trouble with the code because the transition was not applied to the GalleyCard component when I tried to call it multiple times. The problem has now been resolved.
Another problem with the transition was that it disappeared all of a sudden when we lifted the cursor out of the parent division, leaving the onmouseleave listener without a transition. Additionally, this problem has been fixed.
